designer/server/src/test/scala/pl/touk/nussknacker/ui/integration/BaseFlowTest.scala

```diff
@@ -113,17 +113,20 @@ class BaseFlowTest
       "enricher" -> SingleComponentConfig(
         params = Some(
           Map(
-            "param" -> ParameterConfig(Some("'default value'"), Some(StringParameterEditor), None, None, None),
-            "paramDualEditor" -> ParameterConfig(
-              None,
-              None,
-              Some(
-                List(FixedValuesValidator(possibleValues = List(FixedExpressionValue("someExpression", "someLabel"))))
-              ),
-              None,
-              None
+            "param" -> ParameterConfig(
+              defaultValue = Some("'default value'"),
+              editor = Some(StringParameterEditor),
+              validators = None,
+              label = None,
+              hintText = None
+            ),
+            "paramStringEditor" -> ParameterConfig(
+              defaultValue = Some("'default-from-additional-ui-config-provider'"),
+              editor = None,
```

Comment:
This part of the API didn't return the final definition of `Parameter` but only a "UI configuration". It is a mess that someone did because we didn't have definition for base parameters exposed to FE. I'm planning to change it soon.
But regarding this change, I had to change this because this code test `AdditionalUIConfigProvider` and I needed to show that it works on sth other than `validators`